src/perl/Permabit/UserModule.pm

```diff
@@ -37,6 +37,27 @@ sub load {
   return $loaded;
 }
 
+###############################################################################
+# @inherit
+##
+sub loadFromFiles {
+  my ($self) = assertNumArgs(1, @_);
+  my $machine = $self->{machine};
+  my $modFileName = $self->{modFileName};
+
+  # if useUpstream, download vdo package from distribution repo
```

Comment:
A minor point, but you don't really need a comment that just says what the code does. It's only really useful there's something no obvious from the code, such as why we need to do a thing.